containers/buildCentos7Packaging.docker/Dockerfile

```diff
@@ -3,6 +3,10 @@ MAINTAINER Max Neunhoeffer <max@arangodb.com>
 
 RUN yum update -y && yum install -y gcc make rpm-build wget && cd /etc/yum.repos.d && wget http://download.opensuse.org/repositories/shells:fish:release:2/CentOS_7/shells:fish:release:2.repo && yum update -y && yum install -y fish
 
+RUN yum install -y elfutils-libelf-devel git && git clone -b dwz-0.13 git://sourceware.org/git/dwz.git && pushd dwz && ./configure && make && make install && popd
+
```

Comment:
this should be one step, so the docker layering can skip adding the files above that you remove below. Else removing them doesn't have any positive impact.